aip_xx1_launch/launch/radar.launch.xml

```diff
@@ -10,24 +9,16 @@
     <group>
       <push-ros-namespace namespace="front_center"/>
 
-      <!-- socket can -->
-      <group>
-        <include file="$(find-pkg-share pacmod3)/launch/pacmod3.launch.xml">
-          <arg name="use_socketcan" value="true" />
-          <arg name="socketcan_device" value="$(var radar_can_device)" />
-          <arg name="namespace" value="" />>
-        </include>
-      </group>
-
       <!-- tf launch (temporary) -->
-      <node pkg="tf2_ros" exec="static_transform_publisher" name="base_link_to_ars408" output="screen" args="3.8 0.0 0.0 0.0 0.0 0.0 base_link ars408" />
+      <node pkg="tf2_ros" exec="static_transform_publisher" name="base_link_to_ars408" output="screen" args="3.8 0.0 0.0 0.0 0.0 0.0 base_link ars408_front_center" />
```

Comment:
Are you going to move static_tf_publisher to aip_xx1_description? or keep like this?

Reply:
I'll make new PR changing from "static_transform_publisher" to "aip_xx1_description".